### PR TITLE
Make telemetry data accessible in tests

### PR DESCRIPTION
**Pull request by antonyshchenko**
The goal here is to allow checking the number of database calls via tests to make optimization efforts easier to verify and to avoid regressions in the future

### CODE REVIEW

File: integrationtests/src/test/kotlin/xyz/funkybit/integrationtests/api/OrderExecutionTest.kt [hunk 1]
```diff
@@ -1045,6 +1047,13 @@ class OrderExecutionTest : OrderBaseTest() {
 
         makerWsClient.close()
         takerWsClient.close()
+
+        val applyOrderBatchTraceIds = getTraceIdsBySpanName("SequencerRequest.Type.ApplyOrderBatch")
+        assertEquals(3, applyOrderBatchTraceIds.size)
+
+        assertDbCallsCountInTrace(30, applyOrderBatchTraceIds[0])
+        assertDbCallsCountInTrace(31, applyOrderBatchTraceIds[1])
+        assertDbCallsCountInTrace(119, applyOrderBatchTraceIds[2])
```
Comment (antonyshchenko) [inline integrationtests/src/test/kotlin/xyz/funkybit/integrationtests/api/OrderExecutionTest.kt:1056]: Here is the interesting part

File: contracts
Comment (bflood366) [inline contracts:1]: looks like your contracts is not updated or something

**Review by bflood366 — APPROVED** [review]
looks great

File: contracts
Comment (antonyshchenko) [reply contracts:1]: yup, thanks!

**Pull request merged**